auxiliary/ArbitrumGasPriceOracle/cannonfile.toml

```diff
@@ -2,10 +2,10 @@ name = "arbitrum-gas-price-oracle"
 version = "<%= package.version %>"
 description = "Arbiturm Gas Price Oracle Node"
 
-[setting.arbGasPriceOracleAddress]
-defaultValue = "0x000000000000000000000000000000000000006C"
+[var.main]
```

Comment:
I am very much against any breaking changes in naming in configuration declaration at this point.

Reply:
part of the migration process. does not break the naming